docs/src/main/asciidoc/quarkus-runtime-base-image.adoc

```diff
@@ -32,75 +32,69 @@ CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
 == Extending the image
 
 Your application may have additional requirements.
-For example, if you have an application that requires `libfreetype.so`, you need to copy the native libraries to the container.
-In this case, you need to use a multi-stage `dockerfile` to copy the required libraries:
+For example, if you have an application that manipulates graphics, images, or PDFs, you likely have Quarkus AWT extension included in the project and your native executable will require some additional libraries to run.
```

Comment:
Perhaps add a note saying that Quarkus AWT doesn't work with MacOS ARM

Reply:
+1

Reply:
Done